README.rst

```diff
@@ -25,7 +25,7 @@ The Synapse documentation describes `how to install Synapse <https://matrix-org.
 
 Synapse has a variety of `config options
 <https://matrix-org.github.io/synapse/latest/usage/configuration/config_documentation.html>`_
-which can be used to customise its behaviour after installation.
+which can be used to customise its behavior after installation.
```

Comment:
Synapse's authors are largely British, but I don't think we are committed to British English. Though again, I wouldn't consider that grammar (contrary to the PR title).

Reply:
We use British English in all our documentation. 😄 (At least when you keep me away from it.)